tests/phpunit/tests/dependencies/jquery.php

```diff
@@ -87,7 +88,7 @@ public function test_dont_allow_deregister_core_scripts_in_admin() {
 	/**
 	 * @ticket 28404
 	 *
-	 * @covers ::wp_script_is
+	 * @covers ::wp_enqueue_script
```

Comment:
Actually, this one is intending to test `wp_script_is()` as denoted by the test method's name.

I'm relocating this test to a new test class for `wp_script_is()` and including the `@covers` for it. See https://github.com/WordPress/wordpress-develop/pull/4244.

```suggestion
	 * @covers ::wp_script_is
```